Gym_en.rst

```diff
@@ -0,0 +1,213 @@
+gym_hybrid_en.rst
+~~~~~~~~~~~~~~~~~~~
+
+Overview
+=======
+In the gym-hybrid task, the agent's task is simple: Accelerate, turn, or break within a square box of side length 2 to stay in the red target area. The target area is a circle with a radius of 0.1. As shown below.
+
+.. image:: ./images/hybrid.gif
+   :align: center
+
+Install
+====
+
+installation method
+--------
+
+.. code:: shell
+
+    pip install git+https://github.com/thomashirtz/gym-hybrid@master#egg=gym-hybrid
+
+Verify installation
+--------
+
+Method 1: Run the following command, if the version information can be displayed normally, the installation is completed successfully.
+
+.. code:: shell
+
+    pip show gym-hybrid
+
+
+Method 2: Run the following Python program, if no error is reported, the installation is successful.
+
+.. code:: python
+
+    import gym
+    import gym_hybrid
+    env = gym.make('Moving-v0')
+    obs = env.reset()
+    print(obs)
+
+Environment introduction
+=========
+
+action space
+------------
+
+The action space of Gym-hybrid belongs to the mixed space of discrete continuous actions. There are 3 discrete actions: Accelerate, Turn, and Break. The actions Accelerate and Turn need to give the corresponding 1-dimensional continuous parameters.
+
+- \ ``Accelerate (Acceleration value)`` \: Indicates that the agent is accelerated by the size of \ ``acceleration value`` \. The value range of \ ``Acceleration value`` \ is \ ``[0,1]`` \. The numeric type is \``float32``.
+  
+- \ ``Turn (Rotation value)`` \: means to turn the agent in the direction of \ ``rotation value`` \. The value range of \ ``Rotation value`` \ is \ ``[-1,1]`` \. The numeric type is \``float32``.
```

Comment:
- \ ``Turn (Rotation value)`` \: means to turn the agent in the direction of \ ``rotation value`` \. The value range of \ ``Rotation value`` \ is \ ``[-1,1]`` \. The numeric type is \ ``float32``.